pom.xml

```diff
@@ -59,29 +48,483 @@
   </distributionManagement>
 
   <properties>
-    <scm.gitBaseUrl>https://github.com/openhab</scm.gitBaseUrl>
+    <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
+    <project.reporting.outputEncoding>UTF-8</project.reporting.outputEncoding>
+    <oh.java.version>1.8</oh.java.version>
+    <maven.compiler.source>${oh.java.version}</maven.compiler.source>
+    <maven.compiler.target>${oh.java.version}</maven.compiler.target>
+    <maven.compiler.compilerVersion>${oh.java.version}</maven.compiler.compilerVersion>
+
+    <bnd.version>4.1.0</bnd.version>
+    <karaf.version>4.2.1</karaf.version>
+    <sat.version>0.5.0</sat.version>
+    <slf4j.version>1.7.21</slf4j.version>
+    <xtext.version>2.14.0</xtext.version>
   </properties>
 
+  <repositories>
+    <repository>
+      <releases>
+        <enabled>true</enabled>
+        <updatePolicy>never</updatePolicy>
+      </releases>
+      <snapshots>
+        <enabled>false</enabled>
+      </snapshots>
+      <id>jcenter</id>
+      <name>JCenter Repository</name>
+      <url>https://jcenter.bintray.com</url>
+    </repository>
+    <repository>
+      <releases>
+        <enabled>true</enabled>
+        <updatePolicy>never</updatePolicy>
+      </releases>
+      <snapshots>
+        <enabled>false</enabled>
+      </snapshots>
+      <id>esh-3rdparty-mvn</id>
+      <name>openHAB Core :: 3rd party Maven Repository</name>
+      <url>http://eclipse.github.io/smarthome/third-party/m2-repo/</url>
+    </repository>
+    <repository>
+      <id>bintray-maggu2810-maven</id>
```

Comment:
The url to the maggu2810 repository should be removed. Is something like that still needed?

Reply:
It can be removed if you know about a repository that provides the dbus and unix bundle of matthw.
See: https://github.com/openhab/openhab-core/blob/c0ef794924d217d546da29eeda9a739d2cf349b0/bundles/org.openhab.core.io.transport.dbus/pom.xml#L16-L27